embed/embedding.js

```diff
@@ -0,0 +1,148 @@
+/* eslint-disable */
+(function () {
+  init();
+  createIframe();
+})();
+function init() {
+  sb.p = sb.p || {};
+  for (p of sb.s) {
+    sb.css = "../../assets/embed.css";
```

Comment:
embedding script is gonna be stand alone in githubpage are we sure this is gonna work

Reply:
I removed it. I was copied from the original embedding script and I don't think we need it here